server/command.go

```diff
@@ -78,29 +78,29 @@ func executeInstance(p *Plugin, c *plugin.Context, args []string) *model.Command
 	case "select":
 		return executeInstanceSelect(p, c, args)
 	}
-	return responsef("Usage: /jira instance [add,list,select,delete]")
+	return responsef("Please specify a parameter in the form `/jira instance [add,list,select]")
 }
 
 func executeInstanceList(p *Plugin, c *plugin.Context, args []string) *model.CommandResponse {
 	known, err := p.LoadKnownJIRAInstances()
 	if err != nil {
-		return responsef("Failed to load known JIRA instances: %v", err)
+		return responsef("Failed to load known Jira instances: %v", err)
 	}
 	if len(known) == 0 {
 		return responsef("(none installed)\n")
 	}
 
 	current, err := p.LoadCurrentJIRAInstance()
 	if err != nil {
-		return responsef("Failed to load current JIRA instance: %v", err)
+		return responsef("Failed to load current Jira instance: %v", err)
 	}
 
 	keys := []string{}
 	for key := range known {
 		keys = append(keys, key)
 	}
 	sort.Strings(keys)
-	text := "Known JIRA instances (selected instance is **bold**)\n\n| |Key|Type|\n|--|--|--|\n"
+	text := "Known Jira instances (selected instance is **bold**)\n\n| |Key|Type|\n|--|--|--|\n"
```

Comment:
s/Key/URL/ (the "key" is always the URL of the instance, so might as well call it such)

Reply:
ah, this is where we can change to:
```
"Known JIRA instances (selected instance is **bold**)\n\n|Key|URL|Type|\n|--|--|--|\n"
```